RULES/core/context.go

```diff
@@ -453,6 +461,15 @@ func (ctx *context) ninjaFile() string {
 	return ninjaFile.String()
 }
 
+func (ctx *context) RegisterCompDbRule(rule *BuildRule) {
+	ctx.compDbBuildRules[rule.Name] = rule
+}
+
+func (ctx *context) GetCompDbRule(name string) (*BuildRule, bool) {
+	buildRule, ok := ctx.compDbBuildRules[name]
```

Comment:
just return `ctx.compDbBuildRules[name]`

Reply:
I also thought that would work, but turns out that invokes the other method of a map that aborts and then I get:
```
dbt-rules/RULES/core/context.go:469:9: not enough return values
        have (*BuildRule)
        want (*BuildRule, bool)
```